addons/auth_signup/data/mail_template_data.xml

```diff
@@ -341,5 +341,65 @@
             <field name="auto_delete" eval="True"/>
         </record>
 
+        <!-- Email template for new device login alert -->
+        <record id="alert_new_device_email" model="mail.template">
+            <field name="name">Settings: Alert New Device</field>
```

Comment:
Why "settings" in the name ?

Reply:
because it's consistent with the other emails regarding user configuration